language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java

```diff
@@ -69,13 +71,22 @@
     PrintStream out = new PrintStream(bout);
     ImmutableList<Entity> entities =
         ImmutableList.of(
-            new Entity().setName("Larry Page").setSalience(0.426f).setType("PERSON").setMetadata(
-                ImmutableMap.<String, String>builder()
-                    .put("knowledge_graph_mid", "/m/0gjpq")
-                    .put("wikipedia_url", "http://en.wikipedia.org/wiki/index.html?curid=60903")
-                    .build()),
-            new Entity().setName("search engine").setSalience(0.188f).setType("CONSUMER_GOOD"),
-            new Entity().setName("something"));
+            Entity.newBuilder().setName("Larry Page")
+                .setSalience(0.426f)
```

Comment:
Done. Added verbose comments calling out this is mock data based on actual data.